aspnetcore/host-and-deploy/visual-studio-publish-profiles.md

```diff
@@ -25,7 +25,7 @@ The `dotnet new mvc` command produces a project file containing the following ro
 
 The preceding `<Project>` element's `Sdk` attribute imports the MSBuild [properties](/visualstudio/msbuild/msbuild-properties) and [targets](/visualstudio/msbuild/msbuild-targets) from *$(MSBuildSDKsPath)\Microsoft.NET.Sdk.Web\Sdk\Sdk.props* and *$(MSBuildSDKsPath)\Microsoft.NET.Sdk.Web\Sdk\Sdk.targets*, respectively. The default location for `$(MSBuildSDKsPath)` (with Visual Studio 2019 Enterprise) is the *%programfiles(x86)%\Microsoft Visual Studio\2019\Enterprise\MSBuild\Sdks* folder.
 
-`Microsoft.NET.Sdk.Web` (Web SDK) depends on other SDKs, including `Microsoft.NET.Sdk` (.NET Core SDK) and `Microsoft.NET.Sdk.Razor` ([Razor SDK](xref:razor-pages/sdk)). The MSBuild properties and targets associated with each dependent SDK are imported. Publish targets import the appropriate set of targets based on the publish method used.
+`Microsoft.NET.Sdk.Web` ([Web SDK](xref:razor-pages/web-sdk)) depends on other SDKs, including `Microsoft.NET.Sdk` (.NET Core SDK) and `Microsoft.NET.Sdk.Razor` ([Razor SDK](xref:razor-pages/sdk)). The MSBuild properties and targets associated with each dependent SDK are imported. Publish targets import the appropriate set of targets based on the publish method used.
```

Comment:
Should we add a link for the .NET Core SDK references, too? Perhaps one of these:

- https://docs.microsoft.com/en-us/dotnet/core/project-sdk/msbuild-props
- https://docs.microsoft.com/en-us/dotnet/core/sdk

Reply:
Yes, why not add both links?

Reply:
I was just gonna make .NET Core SDK into a link, so I'd have to choose one the URLs.

Reply:
`https://docs.microsoft.com/en-us/dotnet/core/project-sdk/msbuild-props` looks better to me, but either one is fine.